tests/ImageTest.php

```diff
@@ -303,4 +380,23 @@ private function createImage(string $path = null, ImagineInterface $imagine = nu
 
         return new Image($path, $imagine, $filesystem);
     }
+
+    /**
+     * Insert an EXIF header into the passed JPEG data with the specified orientation.
+     */
+    private function addImageOrientation(string $jpegData, int $orientation): string
+    {
```

Comment:
For the next rewiever: there‘s no PHP exif function that allows to write this data.